server/service/src/stock_line/mod.rs

```diff
@@ -0,0 +1,39 @@
+use self::query::{get_stock_line, get_stock_lines};
+
+use super::{ListError, ListResult};
+use crate::{service_provider::ServiceContext, SingleRecordError};
+use repository::{
+    PaginationOption, StockLine, StockLineFilter, StockLineSort, StorageConnectionManager,
+};
+
+// pub mod delete;
+// pub mod insert;
+pub mod query;
+// pub mod update;
+// mod validate;
```

Comment:
remove commented lines...

Reply:
The mutate permission and these and the reason for not popping it into general are all because mutations are coming!
Am just getting the query in first, so that I can update the f/e listview, the implement the modal and then do the mutations 😉